client/apps/shipping-settings/index.js

```diff
@@ -18,7 +18,7 @@ import { mergeHandlers } from 'state/action-watchers/utils';
 import { middleware as rawWpcomApiMiddleware } from 'state/data-layer/wpcom-api-middleware';
 import { combineReducers } from 'state/utils';
 
-export default ( { order_id: orderId, order_href: orderHref, carrier: carrier, continents, carriers, subscriptions } ) => ( {
```

Comment:
No need to alias `carrier: carrier` - renamed `carriers` to `carrier_accounts: carrierAccounts`